Sources/SourceKit/SourceKitServer.swift

```diff
@@ -169,8 +169,13 @@ public final class SourceKitServer: LanguageServer {
         return nil
       }
 
+      #if os(Windows)
+        let pid: Int = unsafeBitCast(GetCurrentProcess(), to: Int.self)
+      #else
+        let pid: Int = Int(getpid())
+      #endif
```

Comment:
Why we are reimplementing things already in Foundation?

This could simply be:

```swift
let pid = Int(ProcessInfo.processInfo.processIdentifier)
```

Which is already cross-platform.

Reply:
Ah fair point, I didn't realize that existed. Plus GetCurrentProcess gives a handle instead of a pid which now that I think about it, is probably the wrong thing to identify a process with. I'll swap it.